### PR TITLE
Feature/specifying non openend processes

### PR DESCRIPTION
**Pull request by lats1**

### CODE REVIEW

File: Resources/views/process/_process_list.html.twig [hunk 1]
```diff
@@ -5,7 +5,11 @@
         <table class="table table-striped table-hover table-borderless">
             <tbody>
             {% for process in processes|slice(0, max_iterations) %}
+                {% if process.visitedByCaseWorker() %}
```
Comment (tuj) [inline Resources/views/process/_process_list.html.twig]: Replace with one liner
<tr{% if not process.visitedByCaseWorker() %} class="not-visited"{% endif %}>

File: Resources/views/process/_process_table_pagination.html.twig [hunk 2]
```diff
@@ -20,7 +20,11 @@
             </thead>
             <tbody>
             {% for process in pagination %}
+                {% if not process.visitedByCaseWorker() %}
```
Comment (tuj) [inline Resources/views/process/_process_table_pagination.html.twig]: Replace with oneliner

File: Resources/views/process/_process_list.html.twig (hunk 1 above)
Comment (lats1) [reply Resources/views/process/_process_list.html.twig]: Fixed in 96f59e1

File: Resources/views/process/_process_table_pagination.html.twig (hunk 2 above)
Comment (lats1) [reply Resources/views/process/_process_table_pagination.html.twig]: Fixed in 3eaf5fc

**Pull request merged**